.github/workflows/ci.yaml

```diff
@@ -24,6 +24,33 @@ jobs:
       - name: Linting - Standardrb
         run: bin/rails standard
 
+  lint-dfe-analytics:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11-alpine
```

Comment:
Can we take the opportunity to migrate to a more recent version? 16 is available